reference/content.yaml

```diff
@@ -712,13 +773,359 @@ components:
         results:
           type: array
           items:
-            $ref: '#/components/schemas/Event'
+            $ref: "#/components/schemas/Event"
         prevPage:
           type: string
         nextPage:
           type: string
         aggregations:
-          - $ref: '#/components/schemas/EventAggregations'
+          - $ref: "#/components/schemas/EventAggregations"
+    Addressables:
+      Article:
+        title: Article
+        description: A reduced piece of editorial content, limited to the parts necessary to render a summary
+        type: object
+        properties:
+          type:
+            type: string
+            description:
+          id:
+            type: string
+            description: The identifier of the article
+          uid:
+            type: string
+            description: The human-readable identifier of the article
+          title:
+            type: string
+            description: The title of the article
+          description:
+            type: string
+            description: A short description of the article's content
+        required:
+          - type
+          - uid
+          - title
+          - description
```

Comment:
Looking at [the type here](https://github.com/wellcomecollection/content-api/blob/main/pipeline/src/types/transformed/addressables.ts#L13), uid and description are optional, id should be required (I think?), same for all content types

Reply:
I can't remember 100% why we made it optional, but we did see instances of it missing it, although it shouldn't really happen anymore. Should we change the type then?

Reply:
I think so, I'll do that